cicd/pipelines/securing-webhooks-with-event-listeners.adoc

```diff
@@ -0,0 +1,23 @@
+[id="securing-webhooks-with-event-listeners"]
+= Securing webhooks with event listeners
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: securing-webhooks-with-event-listeners
+
+toc::[]
+
+Triggers in {pipelines-title} support both HTTP (insecure) and HTTPS (secure HTTP) connections to the `Eventlistener` resource. With the secure HTTPS connection, you get an end-to-end secure connection within and outside the cluster. Enable the secure HTTPS connection, after creating a namespace, for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace. After that, create a `Trigger` resource and a secured route using the re-encrypted TLS termination.
```

Comment:
```suggestion
Triggers in {pipelines-title} support insecure HTTP and secure HTTPS connections to the `Eventlistener` resource. HTTPS secures connections within and outside the cluster. After creating a namespace, you enable HTTPS for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace. After that, you create a `Trigger` resource and a secured route using the re-encrypted TLS termination.
```

Reply:
Consider dropping or modifying line 12 if you think this knowledge is already apparent to your users.